company-profiles/fraudio.md

```diff
@@ -0,0 +1,37 @@
+# Fraudio
+
+## Company blurb
+
+Our startup https://fraudio.com is bringing the latest AI technology to the finance and payments sector and is looking to disrupt it. 
+We use cutting-edge technology and research-driven techniques to create tools that productionize AI for our clients, to combat various illicit activites 
+such as transaction fraud, merchant fraud and money laundering.
+Fraudio is currently comprised of around 25 people from many countries and cultures, which we believe is a strength. 
+We promote transparency, ownership, collaboration, continuous improvement, quality, and having fun while working together. 
+
+We are at a stage where our product is already in production and we are looking to expand it to a full SaaS product.
+
+## Company size
+
+20-30
+
+## Remote status
+
+We are a fully distributed team and all our daily processes are designed for remote work.
+After COVID, we plan to have regular gettogethers with all of us a few times a year, 
+and likely work together in the office for a few days a month for those living nearby.
```

Comment:
Another odd line break

Reply:
Fair enough! I removed the unnecessary line breaks.